content/docs/security/security-overview.md

```diff
@@ -46,9 +46,45 @@ Amazon’s secure data centers backed by [AWS Cloud Security](https://aws.amazon
 
-## Compliance
+## Compliance Overview
 
-Neon has successfully completed audits for SOC 2 Type 1 and Type 2, ISO 27001, and ISO 27701, and adheres to GDPR and CCPA regulations. See [Compliance](/docs/security/compliance).
+At Neon, we implement robust technical controls to secure customer and sensitive data in alignment with SOC2, ISO27001, ISO27701 standards and GDPR and CCPA regulations.
+
+All systems are hosted on AWS, where we have implemented specific security measures to protect data. Below is a detailed breakdown of these compliance-relevant security measures for access control, encryption, network security, event logging, vulnerability management, backups, data deletion and retention:
+
+1. **Customer and Sensitive Data Encryption (AWS KMS)**
+
+   All customer and sensitive data is encrypted using AES-256 encryption at rest. For data in transit, encryption is enforced using TLS 1.2/1.3 protocols across various services. Encryption keys are managed using AWS Key Management Service (KMS) with key rotation policies in place. Only services and users with specific IAM roles can access the decryption keys, and all access is logged via AWS CloudTrail for auditing and compliance purposes.
+
+2. **Fine-Grained Access Control via IAM**
+
+   Access to PII and customer or sensitive data is controlled through AWS Identity and Access Management (IAM) policies. Broad access is limited to the infrastructure and security teams, while other roles operate under least-privilege principles. When additional access needed, access requests to production systems are received via Teleport, where all sessions are recorded. Only managers and on-call personnel are permitted to access production or approve production access requests.
+
+   Additionally, all infrastructure is managed through Terraform, ensuring that any changes to access controls or resources are fully auditable and version-controlled. Regular access reviews and audits are conducted to verify that access rights remain aligned with security best practices.
+
+3. **Data Segmentation and Isolation Using VPCs and Security Groups**
+
+   In our AWS environment, workloads are segmented using Virtual Private Clouds (VPCs) to separate sensitive data environments from other systems. We control network access between services by using security groups and Network Access Control Lists (NACLs), restricting access to only the necessary traffic. This ensures a clear separation of environments, minimizing the risk of unauthorized access or lateral movement between services.
+
+4. **Event-Based Data Anomaly Detection via AWS GuardDuty and Logz.io**
```

Comment:
Do we have an equivalent for Azure?

Reply:
I'll include Azure after the SOC2 audit in November. Let's keep it out for now to prevent confusion. 

https://neondb.slack.com/archives/C03GTPNTTHQ/p1728655452410279?thread_ts=1728637738.939789&cid=C03GTPNTTHQ